app/models/skill.rb

```diff
@@ -0,0 +1,3 @@
+class Skill < ApplicationRecord
```

Comment:
I'm not seeing a model spec for Skill...

Reply:
it should be `skill_spec.rb` seeing it on this end.

Reply:
There are no tests for the skills model. 
Note: We didn't configure simplecov so it is hard to see our test coverage.

Reply:
please add a card in our todo to add simplecov